.npmrc

```diff
@@ -1 +1 @@
-//registry.npmjs.org/:_authToken=${NPM_AUTH}
+//registry.npmjs.org/:_authToken=NPM_AUTH
```

Comment:
don't... leave it as it is

Reply:
In PowerShell, this won't work without the changes i.e. I will have to manually change the shell every time I want to commit or run any npm command in this directory

Reply:
set NPM_AUTH as empty string in env, in your desktop, with your changes ci will break completely